### DISCUSSION TITLE
Disabling resize or maximize of native window

### DISCUSSION THREAD

**Discussion by impubes** [zauberzeug/nicegui#630] [Q&A]
hi,
is there any way to disable resize or maximize of native window?
thank you.

**Comment by falkoschindler** [comment]
Currently that's not possible.
But we're discussing a way to configure the native window over at #640. We're planning to give access to all pywebview settings, so disabling window resizing should be solved this way. Therefore I'll close this discussion in favor of the more general #640.

**Reply by impubes** [reply]
Thank you 👍
